pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -61,6 +59,7 @@ def make_command_wrapper_symlinks(env: MutableMapping[str, str]):
     exist.
     """
     exec_path = Path(__file__).resolve()
+    exec_path.chmod(0o755)
```

Comment:
It's weird to change the permission of the current file in place. It doesn't work without this? It was working before and fundamentally we are not changing anything with respect to calling this file via symlinks.

Reply:
This was added for non-editable installation. We don't need this if we will keep editable install.